packages/locale/typings/js-joda-locale.d.ts

```diff
@@ -10,6 +12,22 @@ export class Locale {
     public equals(other: any): boolean;
 }
 
+export class WeekFields {
+    public static ISO: WeekFields;
+    public static SUNDAY_START: WeekFields;
+    public static ofLocale(locale: Locale): WeekFields;
+    public static of(firstDayOrLocale: core.DayOfWeek | Locale, minDays?: number): WeekFields;
+    public static ofFirstDayOfWeekMinDays(firstDayOfWeek: DayOfWeek, minimalDaysInFirstWeek: number): WeekFields;
```

Comment:
Some of these methods aren't public and shouldn't be typed, in particualr `ofLocale` and `ofFirstDayOfWeekMinDays`.

Also, I think the `of` method could be typed as an overload instead of using an union type. Right now the type accepts a call with a `Locale` _and_ a `number`, which isn't right.